helpers/suggestions/suggestion_creator.py

```diff
@@ -14,7 +14,12 @@ class SuggestionCreator(object):
     def createTeamMediaSuggestion(cls, author_account_key, media_url, team_key, year_str, private_details_json=None):
         """Create a Team Media Suggestion. Returns status (success, suggestion_exists, media_exists, bad_url)"""
 
-        media_dict = MediaParser.partial_media_dict_from_url(media_url.strip())
+        # Sanitize input url
+        media_url = media_url.strip()
+        if '?' in media_url:
```

Comment:
Is there a better way to do this with some existing library? This doesn't deal with `#`